src/contentTypeConfig.js

```diff
@@ -579,9 +579,71 @@ export let contentTypeConfig = {
       `quarter_removed`,
       '"thumbnail": thumbnail.asset->url',
     ]
-  }
+  },
+  "method-v2": [
+    `"id":_id`,
+    `"type":_type`,
+    "brand",
+    `"difficulty":difficulty`,
+    `"published_on":published_on`,
+    `"intro_video": intro_video->{ ${getIntroVideoFields().join(", ")} }`,
+    `child[]->{
+    "resource": ${resourcesField},
+    total_skills,
+    "type":_type,
+    brand,
+    title,
+    "description": ${descriptionField},
+    "thumbnail": thumbnail.asset->url,
+    length_in_seconds,
+    "intro_video": intro_video->{
+      external_id,
+      hlsManifestUrl,
+      video_playback_endpoints
+    },
+    lp_lessons[]->{
+      ${DEFAULT_FIELDS.join(',')}
+    }
+  }`,
+  ],
+  "method-v2-intro-video": [
```

Comment:
I think this should be something like so now:

```suggestion
  "method-v2-intro-video": getIntroVideoFields()
```